edx_credentials_themes/templates/edx.org/credentials/programs/micromasters/certificate.html

```diff
@@ -1,34 +1,38 @@
 {% extends 'edx.org/credentials/programs/base.html' %}
 {% load i18n %}
+{% load html %}
 
 {% block message %}
-  {# Translators: MicroMasters should not be translated #}
-  {% blocktrans trimmed with organization_name=program_details.organizations.0.key %}
-    You worked hard to earn your MicroMasters Credential from {{ organization_name }}.
-  {% endblocktrans %}
+  {% filter htmlescape  %}
```

Comment:
Nit: I think there's an extra space between `htmlescape` and `%}`.

Reply:
👍